rewrite-java/src/test/java/org/openrewrite/java/ModifierTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class ModifierTest implements RewriteTest {
```

Comment:
Parser tests have to go in **rewrite-java-tck** (which is a module that's executed by the java-8, java-11, java-17 and java-21 modules). In this case these tests should be added to the [AnnotationTest](https://github.com/openrewrite/rewrite/blob/a1d887038c4c67d9f07801200f0a20afec0bb9a3/rewrite-java-tck/src/main/java/org/openrewrite/java/tree/AnnotationTest.java#L39) class instead of creating a new test class.

Reply:
After moving the tests to above module test were failing for java 8 and 11. I'm not able to debug the code as most of the code dependent on java 17 that causing issues in creating parser for 8 and 11. Any suggestions?

Reply:
I have moved the tests to `AnnotationTest`.